Scripts/extras/pull_and_build_from_git.py

```diff
@@ -779,6 +779,19 @@ def check_build_keys(self, keys_to_check):
 
         return False
 
+    def copy_extra_files(self):
+        """
+        Copies any extra files specified in the build config into the destination folder for packaging.
+        """
+        extra_files_to_copy = self.platform_config.get('extra_files_to_copy', [])
+
+        for (source, dest) in extra_files_to_copy:
+            print(f"Source file: {self.base_folder / source}, Destination file: {self.package_install_root / dest}")
+            shutil.copy2(
+                self.base_folder / source,
+                self.package_install_root / dest
```

Comment:
This  needs error checking and auto-failure if the copy2 fails.  Or does it throw an uncaught exception (which would be okay)?  Maybe try it with a file that doesn't exist and see what happens, we want the package to 'fail'

Reply:
The 2nd one--it will throw a FileNotFoundError which will cause the package to fail.